README.md

```diff
@@ -1,77 +1,373 @@
+#### Make Directory
 
-<h2 id="javascript">JavaScript</h2>
+We want a place where we can put all our work in. While in the terminal make a new directory called VetsWhoCode by typing mkdir in Windows or MacOS and press enter:
 
-JavaScript is where you go from being an artist to being a wizard in web development. A deep knowledge of JavaScript fundamentals will make you a highly employable developer. Complete Javascript Info to get a good beginner grasp of JS.
+`mkdir VetsWhoCode`
 
-JavaScript is derived from ECMAScript. This is important since new versions come out having major impact on JavaScript. Read through this resource to understand the newest version of ECMAScript:
+#### Change Working Directory
 
+Now that we have made a new directory lets go there where we will put our work on our machine. te do that by in both windows and MacOS we do this by dyping cd then pressing enter:
 
+`cd VetsWhoCode`
 
-[Javascript Info](http://javascript.info/)
 
+#### Change to previous Directory
+If you want to go back to the previous directory for both Windows and MacOS you type this and press enter:
 
+`cd ..`
 
-<h2 id="capstone">Capstone</h2>
+#### Delete a directory
+Let's say you make a typo and need to delete a directory. Here are the commands to do that.
 
-Build a website telling about you and include your favorite things and then push it to GitHub. Bonus points if you use [GitHub Pages](https://pages.github.com/) or [surge.sh](https://surge.sh/) to host your site.
+MacOS; `rm -r NameOfDirectory`
+Windows: `rmdir /S NameOfDirectory`
+
+### Code Editor
+
+An Integrated Development Environment (IDE) is a software application that provides comprehensive tools and features to assist developers in writing, debugging, and deploying software. An IDE typically includes a code editor, a debugger, a compiler or interpreter, and other tools such as a profiler, a version control system, and a build automation tool.
+An IDE can also provide features like auto-completion, syntax highlighting, and code refactoring to make coding faster and more efficient. IDEs are commonly used for developing software applications, web applications, and mobile apps. Our preferred IDE is Visual Studio Code. Download it [here](https://code.visualstudio.com/). You can also install our developer extension pack by going to extensions and typing [VetsWhoCode Extension Pack](https://marketplace.visualstudio.com/items?itemName=VetsWhoCode.vetswhocode-extension-pack).
+
+### Browser
+
+The preferred browser for Vets Who Code is the Google Chrome, due to its popularity and its robust developer tools. To download it please go [here](https://www.google.com/chrome/) and follow the prompts.
+
+
+### Using GitHub
+ We use GitHub extensively here. Below you’ll find the steps to use it within the terminal or command prompt.
+
+1.	 Install Git: Git is a version control system used by GitHub. You can download and install Git for Windows from the official Git website.
```

Comment:
I thought we preferred the Github CLI over `git` cli. Also, what's our opinion on Github desktop?